cmd/zetatool/filterdeposit/evm.go

```diff
@@ -0,0 +1,208 @@
+package filterdeposit
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"math/big"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/nanmu42/etherscan-api"
+	"github.com/spf13/cobra"
+	"github.com/zeta-chain/protocol-contracts/pkg/contracts/evm/erc20custody.sol"
+	"github.com/zeta-chain/protocol-contracts/pkg/contracts/evm/zetaconnector.non-eth.sol"
+	"github.com/zeta-chain/zetacore/cmd/zetatool/config"
+)
+
+const (
+	TopicsDeposited = 2
+	TopicsZetaSent  = 3
+	DonationMessage = "I am rich!"
```

Comment:
We already have the constant defined in the `common` package, we should reuse it